src/Schema/CMS/Events/BatchEditFlow.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * Schemas describing CMS BatchEditFlow events
+ * @packageDocumentation
+ */
+
+import { CmsContextModule } from "../Values/CmsContextModule"
+import { CmsOwnerType } from "../Values/CmsOwnerType"
+
+/**
+ * A partner has seen the max edit limit reached tool tip.
+ *
+ * @example
+ * ```
+ * {
+ *   action: "shownMaxEditLimitReachedTooltip",
+ *   context_module: "batchEditFlow",
+ *   context_page_owner_type: "batchEdit",
+ *   context_page_owner_id: "67b646ecbe87376bfeb3f962",
+ *   referrer: ""
+ * }
+ * ```
+ */
+export interface CmsShownMaxEditLimitReachedTooltip {
+  action: "shownMaxEditLimitReachedTooltip"
```

Comment:
We'll want to encode that in here: https://github.com/artsy/cohesion/blob/05f95097b2a84651b249b602e8496a9d496adf61/src/Schema/CMS/Events/index.ts#L25